app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,11 @@
+<!-- Footer -->
+<footer class="page-footer font-small white">
+
+  <!-- Copyright -->
+  <div class="footer-copyright text-center py-3">© 2018 Copyright:
```

Comment:
Make the date be dynamic to the current year. 

Add footer menu on the right hand side for footer links, like about, terms and conditions, etc.